packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -545,6 +593,12 @@ export class GarbageCollector implements IGarbageCollector {
         );
     }
 
+    public dispose(): void {
+        if (this.sessionExpiryTimeout !== undefined) {
```

Comment:
nit: This check is not needed. clearTimeout silently does nothing is undefined is passed.